test_requirements/django_17.txt

```diff
@@ -0,0 +1,5 @@
+-r base.txt
+Django>=1.7,<1.8
+django-discover-runner
```

Comment:
don't think that you need discover runner,  should be already a part of Django itself.